storm-core/src/clj/backtype/storm/util.clj

```diff
@@ -31,6 +33,8 @@
   (:import [java.io File FileOutputStream RandomAccessFile StringWriter
             PrintWriter BufferedReader InputStreamReader IOException])
   (:import [java.lang.management ManagementFactory])
+  (:import [java.nio.file Files Paths])
+  (:import [java.nio.file.attribute FileAttribute])
```

Comment:
These two imports are duplicated.  Also, it would be nice to clean up the `java.nio.file` imports.  I see us importing Paths twice even without this change.